build_container/build_container_windows.ps1

```diff
@@ -135,8 +143,8 @@ AddToPath C:\tools\nasm-$nasmVersion
 
 # Python3 (do not install via msys2 or the MS store's flavors, this version follows Win32 semantics)
 DownloadAndCheck $env:TEMP\python3-installer.exe `
-                 https://www.python.org/ftp/python/3.9.0/python-3.9.0-amd64.exe `
-                 fd2e2c6612d43bb6b213b72fc53f07d73d99059fa72c96e44bde12e7815073ae
+                 https://www.python.org/ftp/python/3.9.4/python-3.9.4-amd64.exe `
```

Comment:
PR description mentions python 3.9.5

Reply:
Good catch TY

Reply:
3.9.4 is the latest version (released today).